README.md

```diff
@@ -13,13 +13,15 @@ provisioning VMs, and [OpenShift Origin](https://github.com/openshift/origin) fo
 - [Installation](#installation)
   - [Prerequisites](#prerequisites)
   - [Installing Minishift](#installing-minishift)
-    - [Installing via Homebrew](#installing-via-homebrew)
+    - [Installing with Homebrew](#installing-with-homebrew)
 - [Quickstart](#quickstart)
   - [Starting Minishift](#starting-minishift)
   - [Deploying a sample application](#deploying-a-sample-application)
   - [Reusing the Docker daemon](#reusing-the-docker-daemon)
 - [Documentation](#documentation)
 - [Known Issues](#known-issues)
+  - [Bugs](#bugs)
+  - [Unsupported features](#unsupported-features)
```

Comment:
In the beginning I myself suggested this but was pushed back because (I can't remember who said this) these limitations or known issues were supposedly important for the users to know before they get started. In general, once we have a publication flow, we should move these to a more appropriate place depending on what they represent.